src/main/java/ru/postgrespro/perf/pgmicrobench/statanalyzer/DataLoader/Loader.java

```diff
@@ -0,0 +1,19 @@
+package ru.postgrespro.perf.pgmicrobench.statanalyzer.DataLoader;
+
+import java.util.Arrays;
+
+public class Loader {
+    private double[] latencies;
```

Comment:
может стоит это поле на List<Double> поменять?
используется только в классе ModalityDetectorTests и там стримом переводится из листа в массив, а если сразу сделать листом то переводить не придется
```suggestion
    private final List<Double> latencies;
```